frontend/api/auth/index.ts

```diff
@@ -0,0 +1,217 @@
+import passport from 'passport';
+import { sign } from 'jsonwebtoken';
+import express, { Express, Request, Response, NextFunction } from 'express';
+import { Strategy as FacebookStrategy } from 'passport-facebook';
+import { Strategy as GoogleStrategy } from 'passport-google-oauth20';
+import { Strategy as TwitterStrategy } from 'passport-twitter';
+import type { Profile } from 'passport';
+import { prisma } from '../../src/server/prisma';
+import { assignAnswerToUser } from '../answers';
+import ms from 'ms';
+import { respond404 } from '../../src/server/errors';
+
+const app: Express = express();
+
+if (process.env.VERCEL_ENV === 'preview') {
+  process.env[
+    'BASE_URL'
+  ] = `https://volebni-kalkulacka-2022-git-${process.env.VERCEL_GIT_COMMIT_REF}-ceskodigital.vercel.app`;
```

Comment:
```suggestion
  ] = process.env.VERCEL_URL;
```

Reply:
VERCEL_URL je URL commitu, pokud se nepletu. My bychom chteli URL branche, protoze ta je registrovana pro `staging` v OAutech.

Reply:
Jo, máš pravdu. Pak to tam může být rovnou hardcoded, protože to nebude fungovat pro ty ostatní branche. Ještě něco zkusím.